activejob/lib/active_job/test_helper.rb

```diff
@@ -192,27 +192,13 @@ def assert_performed_jobs(number, only: nil)
     #     end
     #   end
     #
-    # The block form supports filtering. If the :only option is specified,
-    # then only the listed job(s) will be performed.
-    #
-    #     def test_hello_job
-    #       assert_performed_jobs 1, only: HelloJob do
-    #         HelloJob.perform_later('jeremy')
-    #         LoggingJob.perform_later
-    #       end
-    #     end
-    #
-    # An array may also be specified, to support testing multiple jobs.
+    # It can be asserted that no jobs of a specific kind are performed:
```

Comment:
@y-yagi we can expand similar to above deleted documentation, to specify which option is used to perform the filtering.